File: Bookworm/app/src/main/java/com/briatka/pavol/bookworm/customobjects/BookObject.java
package com.briatka.pavol.bookworm.customobjects;

import org.simpleframework.xml.Element;
import org.simpleframework.xml.Root;


@Root(name = "GoodreadsResponse", strict = false)
public class BookObject {


    @Element(name = "book")
    private Book book;


    public Book getBook() {
        return book;
    }

}

File: Bookworm/app/src/main/java/com/briatka/pavol/bookworm/customobjects/BookObject.kt
package com.briatka.pavol.bookworm.customobjects

import org.simpleframework.xml.Element
import org.simpleframework.xml.Root


@Root(name = "GoodreadsResponse", strict = false)
class BookObject  @JvmOverloads constructor(
        @field:Element(name = "book")
        @param:Element(name = "book")
        val book: Book? = null
) {}